mask/eolearn/mask/cloud_mask.py

```diff
@@ -389,96 +380,103 @@ def _do_single_temporal_cloud_detection(self, bands):
                 self.mono_classifier, mono_features
             )
 
-        return mono_proba[..., np.newaxis]
+        return mono_proba[..., None]
 
-    def _do_multi_temporal_cloud_detection(self, bands, is_data, sigma):
+    def _do_multi_temporal_cloud_detection(self, bands: np.ndarray, is_data: np.ndarray, sigma: float) -> np.ndarray:
         """Performs a cloud detection process on multiple scenes at once"""
         multi_proba = np.empty(np.prod(bands.shape[:-1]))
         img_size = np.prod(bands.shape[1:-1])
 
         n_times = bands.shape[0]
 
-        loc_mu = None
-        loc_var = None
-
-        prev_nt_min = None
-        prev_nt_max = None
+        local_avg, local_var = None, None
+        prev_min_tdx, prev_max_tdx = None, None
 
-        for t_i in range(n_times):
+        for tdx in range(n_times):
```

Comment:
what is a tdx? a time index? the loop body is long, and this seems like an important part of it, a longer name seems appropriate

Reply:
`idx` is standard for an index, so I opted for `tdx` as sort of a time index. I can make it longer, but it's also used as an appendix in a lot of variables.

Reply:
- renamed to `t_idx`
- used `left` and `right` for window min/max indices, i think it makes more sense. if not, we can discuss better names in a live session